clouddriver-kubernetes/src/main/java/com/netflix/spinnaker/clouddriver/kubernetes/model/KubernetesV2JobStatus.java

```diff
@@ -1,7 +1,7 @@
 /*
- * Copyright 2019 Armory
+ * Copyright 2020 Google, Inc.
```

Comment:
At this point I'm assuming this was the editor making these changes?

Reply:
Ugh, yeah I think my IDE was overambitious here...this will be fun to fix.

Reply:
Just came across one instance of a file you wrote but ended up with the Netflix header, so I'm also changing that to correct history 👼